### DISCUSSION TITLE
How to apply material to DXF imported section?

### DISCUSSION THREAD

**Discussion by garyfairbairn** [robbievanleeuwen/section-properties#270] [Q&A] [answered]
How can I apply a material to a DXF imported section?

**Comment by robbievanleeuwen** [comment, marked as answer]
Hi @garyfairbairn, apologies for the delayed response. At the moment, the best way to do this to apply the material to the ``Geometry`` or ``CompoundGeometry`` object after importing the dxf.

E.g. for a ``Geometry`` object:

```python
geom = from_dxf(dxf_filepath="example.dxf")
geom.material = my_material
```
